src/propIs.js

```diff
@@ -0,0 +1,28 @@
+var _curry3 = require('./internal/_curry3');
+var is = require('./is');
+var propSatisfies = require('./propSatisfies');
+
+
+/**
+ * Returns `true` if the specified object property is of the given type;
+ * `false` otherwise.
+ *
+ * @func
+ * @memberOf R
+ * @category Type
+ * @sig Type -> String -> Object -> Boolean
+ * @param {Function} type
+ * @param {String} name
+ * @param {*} obj
+ * @return {Boolean}
+ * @see R.is
+ * @see R.propSatisfies
+ * @example
+ *
+ *      R.propIs('x', Number, {x: 1, y: 2});  //=> true
+ *      R.propIs('x', Number, {x: 'foo'});    //=> false
+ *      R.propIs('x', Number, {});            //=> false
+ */
+module.exports = _curry3(function propIs(type, name, obj) {
+  return propSatisfies(is(type), name, obj);
+});
```

Comment:
> which is our only option as things stand.

Unless we count this:

    useWith(propSatisfies, is);

(and why is that cheating?)

But my question is, as we're talking about simplifying the APIs of `converge` and `useWith`, would we really need all the flexibility implied here?  Is there any simpler convention that we could assume?  Perhaps not, as I'm not coming up with anything right away, but it would be really nice if this was not too complex.